build/Python/Makefile

```diff
@@ -1,10 +1,10 @@
 
-PYTHONVERSION=/usr/include/python2.7
-PYTHONLIB=-lpython
+PYTHONVERSION=/usr/include/python3.5
```

Comment:
Don't worry! I will modify this makefile later when I finish with the Python part